parsers/fortran.c

```diff
@@ -582,10 +582,11 @@ static void makeLabelTag (vString *const label)
 
 static lineType getLineType (void)
 {
-	vString *label = vStringNew ();
+	static vString *label = NULL;
```

Comment:
Maybe in CTags specifically, but not in general. Sometimes, like for immutable global data (ex. lookup tables) leaving reachable leaks can speed up program shutdown, but in this case it's just for a small optimization that could probably be achieved better in other ways (ex. stack allocation) and would be quite problematic if CTags and/or the parsers were ever (made to be) used in any kind of multi-threaded scenario.